api/managementpb/backup/backups.proto

```diff
@@ -58,12 +62,130 @@ message RestoreBackupResponse {
   string restore_id = 1;
 }
 
+// ScheduledBackup represents scheduled task for backup.
+message ScheduledBackup {
+  // Machine-readable ID.
+  string scheduled_backup_id = 1;
+  // Machine-readable service ID.
+  string service_id = 2;
+  // Service name.
+  string service_name = 3;
+  // Machine-readable location ID.
+  string location_id = 4;
+  // Location name.
+  string location_name = 5;
+  // How often backup will be run in cron format.
+  string cron_expression = 6;
+  // First backup wouldn't happen before this time.
+  google.protobuf.Timestamp start_time = 7;
+  // Artifact name.
+  string name = 8;
+  // Description.
+  string description = 9;
+  // Retry mode.
+  RetryMode retry_mode = 10;
+  // Delay between each retry. Should have a suffix in JSON: 1s, 1m, 1h.
+  google.protobuf.Duration retry_interval = 11;
+  // How many times to retry a failed backup before giving up.
+  uint32 retry_times = 12;
+  // If scheduling is enabled.
+  bool enabled = 13;
+  // Backup data model.
+  DataModel data_model = 14;
+  // Database vendor e.g. PostgreSQL, MongoDB, MySQL.
+  string vendor = 15;
+  // Last run.
+  google.protobuf.Timestamp last_run = 16;
+  // Next run.
+  google.protobuf.Timestamp next_run = 17;
+}
+
+message ScheduleBackupRequest {
+  // Service identifier where backup should be performed.
+  string service_id = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  // Machine-readable location ID.
+  string location_id = 2 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  // How often backup should be run in cron format.
+  string cron_expression = 3 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  // First backup wouldn't happen before this time.
+  google.protobuf.Timestamp start_time = 4;
+  // Name of backup.
+  string name = 5;
+  // Human-readable description.
+  string description = 6;
+  // Retry mode.
+  RetryMode retry_mode = 7;
```

Comment:
I think we can discard this field .
Retry_times == 0 => Retry Mode = MANUAL
Retry_times > 0 => Retry Mode = AUTO

Whata do you think @atymchuk

Reply:
We removed retries entirely for now.